setup.py

```diff
@@ -160,8 +159,6 @@ def run(self):
     "dropbox": [
         "dropbox>=2.1.0"
     ],
-    "elasticsearch": [
```

Comment:
you should keep it for empty for few versions not to break overlays depending on Invenio.